cmd/airflow_test.go

```diff
@@ -224,6 +217,72 @@ func (s *AirflowSuite) mockUserInput(i string) (r, stdin *os.File) {
 }
 
 func (s *AirflowSuite) TestAirflowInit() {
+	TemplateList = func() ([]string, error) { return []string{"A", "B", "C", "D"}, nil }
+	s.Run("initialize template based project via select-template flag", func() {
+		cmd := newAirflowInitCmd()
+		cmd.Flag("name").Value.Set("test-project-name")
+		cmd.Flag("from-template").Value.Set("")
```

Comment:
nit: I don't feel like this test case is actually triggering the template selection code path, could we add a bit more robust check to validate that some/any files are correctly created based on the template selection value. Maybe just set it to "select-template". 

Saying it based on the codecov report 😅 (https://app.codecov.io/gh/astronomer/astro-cli/pull/1728)